lib/db.js

```diff
@@ -133,7 +160,8 @@ function _bibSqlQueryString (options) {
         ) _BS
     ) as bib_statements,
     (
-      SELECT ${_jsonAggSelectString('_IS', { additionalColumns: { s: 'subject_id', bn: 'blanknode' } })} AS statements
+      SELECT 
+        ${_jsonAggSelectString('_IS', { additionalColumns: { s: 'subject_id', bn: 'blanknode' } })} FILTER (WHERE AS statements
```

Comment:
Is that `FILTER (WHERE` supposed to be there?

Reply:
thanks for catching that! I was considering implementing the filter in the query when I realized that it could be done outside of the SQL and switched approaches, but forgot to remove this (of course the one time I didn't git diff before adding a file is when something like this is in there)